src/components/article_swipe/ArticleSwipeControl.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Bar, DismissButton, OpenButton, SaveButton, ButtonInner } from "./ArticleSwipeControl.sc.js";
+import { white, darkGrey, orange600 } from "../colors";
+import SwipeIcon from "./SwipeIcon.js";
+
+export default function ArticleSwipeControl({ onDismiss = () => {}, onOpen = () => {}, onSave = () => {}, children }) {
+    return (
+        <Bar>
+            <DismissButton onClick={onDismiss} aria-label="Dismiss">
+                <ButtonInner>
+                    <SwipeIcon name="dismiss" color={darkGrey} size={22} />
```

Comment:
If we used the <CloseRoundedIcon> directly it would be easier to read I think. Do we really need the extra indirection given by the `SwipeIcon`?